5-Ansible_Deep_Dive/roles/user/handlers/main.yml

```diff
@@ -0,0 +1,33 @@
+# handlers file for roles/user
+
+- name: Add new user to sudoers
+  lineinfile:
```

Comment:
Are you sure this should be a handler?

Reply:
The idea was to play in only if new user is created

Reply:
Ok